main.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"fmt"
+	"sniffer/internal/config"
+	"sniffer/internal/validator"
+	"sniffer/internal/version"
+	"sniffer/pkg/accumulator"
+	"sniffer/pkg/conthandler"
+
+	logger "github.com/kubescape/go-logger"
+	helpers "github.com/kubescape/go-logger/helpers"
+)
+
+func waitOnCacheAccumulatorErrorCode(cacheAccumulatorErrorChan chan error) {
+	err := <-cacheAccumulatorErrorChan
+	if err != nil {
+		logger.L().Fatal("", helpers.String("ebpf engine failed on error ", fmt.Sprintf("%v", err)))
+	}
+}
+
+func startingOperations() error {
+	return nil
+}
+
+func main() {
+	configData, err := config.GetConfigurutionData()
```

Comment:
[Question]: Where is this function? Do not push code that does not compile.

Reply:
done